storybook/.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
```

Comment:
This will be picked up by storybook – for some reason it looks like it was trying to pick up the root `babel.config.js` and not showing stories

Reply:
HT @bertya